backend/src/apiserver/visualization/requirements.txt

```diff
@@ -14,3 +14,5 @@ tensorflow-metadata==0.21.1
 tensorflow-model-analysis==0.21.5
 tensorflow-data-validation==0.21.1
 tornado==6.0.2
+# Increase google-api-core to some release AFTER 1.21.0 when it becomes available
```

Comment:
nit: I think it will be valuable directly putting this line here as comment, because it explained more context.
```
Fix the google-api-core to 1.16.0 for backend visualization server because of a regression of google-api-core 1.21.0 (googleapis/python-pubsub#115)
```